src/components/submission/decrypt.vue

```diff
@@ -21,27 +21,36 @@ except according to the terms contained in the LICENSE file.
             {{ $t('exportOptions') }}
           </p>
           <div class="checkbox"
-            :class="{ disabled: splitSelectMultiplesTitle != null }">
-            <label :title="splitSelectMultiplesTitle">
+            :class="{ disabled: splitSelectMultiplesDescription != null }">
+            <label v-tooltip.sr-only>
               <input v-model="splitSelectMultiples" type="checkbox"
-                :disabled="splitSelectMultiplesTitle != null">
+                :disabled="splitSelectMultiplesDescription != null"
+                :aria-describedby="splitSelectMultiplesDescribedBy">
               {{ $t('field.splitSelectMultiples') }}
             </label>
+            <p v-if="splitSelectMultiplesDescription != null"
+              id="submission-download-split-select-multiples-disabled"
+              class="sr-only">
+              {{ splitSelectMultiplesDescription }}
+            </p>
           </div>
```

Comment:
This is a more complicated case, because we want `aria-describedby` on the `<input>`, but we want the tooltip on the wrapping `<label>`. It turns that `v-tooltip.sr-only` works well for this use.